ustriage/ustriage_test.py

```diff
@@ -45,6 +45,7 @@ def test_auto_date_range_weekend(today, keyword):
     ('2019-05-07', '2019-05-06', None),  # reverse range definition
     ('2019-05-06', '2019-05-09', None),  # more than two days apart
     ('2019-05-18', '2019-05-18', None),  # Saturday
+    ('2021-04-16', '2021-04-16', None),  # Friday
```

Comment:
While touching it, should we complete it by adding a "Sunday" case and maybe a "Thursday/Friday triage" as well?

Reply:
Absolutely, but I feel that's best in a separate PR. I'll have a follow-up to this. Thanks.

Reply:
If you'll do the follow up, then ok.
But the lessons learned usually is that those "extra PRs later" never happen.

Reply:
Lol, opened #26 🎉